pkg/multicluster/local/local_cluster_manager.go

```diff
@@ -19,31 +19,25 @@ type ClusterManager struct {
 }
 
 func (cm *ClusterManager) GetClusters() ([]multicluster.Cluster, error) {
-	var clusters []multicluster.Cluster
-	cluster, err := cm.GetLocalCluster()
-	if err != nil {
-		return clusters, err
-	}
-	clusters = append(clusters, cluster)
-	return clusters, nil
-}
-
-func (cm *ClusterManager) GetLocalCluster() (multicluster.Cluster, error) {
 	clusterMetadataConfigmap := corev1.ConfigMap{}
 	namespacedName := client.ObjectKey{
 		Name:      clusterMetadataConfigmapName,
 		Namespace: cm.Namespace,
 	}
 	if err := cm.Client.Get(context.Background(), namespacedName, &clusterMetadataConfigmap); err != nil {
-		return multicluster.Cluster{}, err
+		wrappedError := fmt.Errorf("Error in GetClusters: %w", err)
```

Comment:
We sometimes use this method and sometimes errors.Wrap. If you have a preference let's document it in the contribution guidelines and stick to one option.

Reply:
I've reverted this file to the master branch, it is not a part of this PR :-)